lib/usd/hdMaya/delegates/sceneDelegate.cpp

```diff
@@ -28,24 +28,24 @@
 #include <pxr/imaging/hd/rprim.h>
 #include <pxr/imaging/hd/tokens.h>
 
+#include <pxr/imaging/hd/light.h>
 #include <pxr/imaging/hdx/renderSetupTask.h>
 #include <pxr/imaging/hdx/renderTask.h>
 #include <pxr/imaging/hdx/tokens.h>
 
-#include <maya/MDGMessage.h>
 #include <maya/MDagPath.h>
 #include <maya/MDagPathArray.h>
+#include <maya/MDGMessage.h>
```

Comment:
I always waffle on this, whether to order capital letters before lower case, or to sort case-insensitively. :/

Reply:
I hear you. I honestly don't have a strong preference. I can change it if you guys prefer it the other way any-times.

Reply:
Yeah, I don't have a strong preference either. Just maybe a justification as to why it may have been that way. :)

Reply:
I want to go with whatever clang-format does by default for this, since I'd eventually like to start using that to handle header ordering automatically.  I wonder if maybe we should just create an initial `.clang-format` file, which disables everything EXCEPT the header ordering, and then just run that on the whole repo?  That way we can at least start having that be standardized, before we come to agreement on all the other nitty-gritty style details.